core/sql/executor/ExHbaseIUD.cpp

```diff
@@ -4193,7 +4188,7 @@ ExWorkProcRetcode ExHbaseAccessSQRowsetTcb::work()
 	      step_ = DONE;
 	  }
 	  break;
-      case ASYNC_INSERT_COMPLETE:
+      case ASYNC_OPERATION_COMPLETE:
```

Comment:
I can change it Hans. Also, I wanted to confirm if the stats returned is correct. So, I might do another code push